### ISSUE TITLE
Crash - Konfetti NullPointerException

### ISSUE THREAD

**Issue by ouchadam** [vector-im/element-android#5925]
The Konfetti v2 upgrade has introduced a regression on android versions 21/22

```
Caused by: java.lang.NullPointerException: 
  at nl.dionsegijn.konfetti.xml.KonfettiView.onVisibilityChanged (KonfettiView.kt:3)
  at android.view.View.dispatchVisibilityChanged (View.java:9306)
  at android.view.View.setFlags (View.java:10737)
  at android.view.View.<init> (View.java:4410)
  at android.view.View.<init> (View.java:3891)
  at android.view.View.<init> (View.java:3870)
  at nl.dionsegijn.konfetti.xml.KonfettiView.<init> (KonfettiView.kt:1)
```

| Occurrences (30 days) | Total Occurrences | Impacted users (lifetime) |
| --- | --- | --- |
| 100 | 113 | 15 |

https://github.com/DanielMartinus/Konfetti/issues/297

**Comment by ouchadam** [comment]
Upgraded severity to critical due to this bug stopping users on API 21/22 from entering the timeline

**Comment by ouchadam** [comment]
Fixed in 1.4.14
